src/lib.rs

```diff
@@ -5,10 +5,14 @@ Defines basic math types useful for computer graphics.
 Designed to serve as an interoperability standard between libraries.
 */
 #![no_std]
-#![deny(missing_docs)]
-
-#[cfg(feature = "serde")]
-extern crate serde;
+#![deny(
+    missing_docs,
+    rust_2018_compatibility,
```

Comment:
why are these needed to be denied? the PR is using rust2018, so naturally it has to be compatible or it will not compile.

Reply:
If i remember there is one lint in rust_2018_compatibility that is not use by default and rust_2018 idioms ensure the code follow some styling thing that could be remove in rust 2021.